src/test/java/com/debatetimer/controller/poll/PollControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package com.debatetimer.controller.poll;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.debatetimer.controller.BaseControllerTest;
+import com.debatetimer.domain.member.Member;
+import com.debatetimer.domain.poll.PollStatus;
+import com.debatetimer.domain.poll.VoteTeam;
+import com.debatetimer.dto.poll.response.PollInfoResponse;
+import com.debatetimer.entity.customize.CustomizeTableEntity;
+import com.debatetimer.entity.poll.PollEntity;
+import io.restassured.http.ContentType;
+import io.restassured.http.Headers;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+class PollControllerTest extends BaseControllerTest {
+
+    @Nested
+    class CreatePoll {
+
+        @Test
+        void 선거를_생성할_수_있다() {
+            Member member = memberGenerator.generate("email@email.com");
+            CustomizeTableEntity table = customizeTableGenerator.generate(member);
+            Headers headers = headerGenerator.generateAccessTokenHeader(member);
+
+            given()
+                    .contentType(ContentType.JSON)
+                    .headers(headers)
+                    .pathParam("tableId", table.getId())
+                    .when().post("/api/polls/{tableId}")
+                    .then().statusCode(HttpStatus.CREATED.value());
+        }
```

Comment:
- 아래와 같이 안한 이유가 있나요? (아래 코드가 마음에 든다면, 다른 PollControllerTest, PollDocumentTest 에 반영 부탁드립니다)
```suggestion
        @Test
        void 선거를_생성할_수_있다() {
            Member member = memberGenerator.generate("email@email.com");
            CustomizeTableEntity table = customizeTableGenerator.generate(member);

            given()
                    .contentType(ContentType.JSON)
                    .header(HttpHeaders.AUTHORIZATION, accessToken())
                    .pathParam("tableId", table.getId())
                    .when().post("/api/polls/{tableId}")
                    .then().statusCode(HttpStatus.CREATED.value());
        }
```

Reply:
[의견]

이건 일단 보류, CustomizeControllerTest에서 작성한 방식으로 맞춘거라 만약 필요하다면 전부 개선할 필요가 있음

1. 헤더에 넣는 값을 명확히 한다는 점에서 좋으나,
2. 반복되는 코드를 감추려고 headerGenerator를 쓰는 거고 의미상으로 안읽히진 않아서 지금도 나쁘지 않다고 생각